### ISSUE TITLE
Upload progress bar contrast should be higher

### ISSUE THREAD

**Issue by josephrocca** [huggingface/huggingface_hub#554]
Here's the current progress bar:
![image](https://user-images.githubusercontent.com/1167575/146874320-bfaf7649-4783-4beb-936b-bb79fd83dd19.png)

For better accessibility and UX I think it should be closer to something like this:
![image](https://user-images.githubusercontent.com/1167575/146874482-9ca91a63-776d-45dd-bdfb-78150df636e0.png)
